Dockerfile

```diff
@@ -554,3 +554,8 @@ ADD patches/tensorboard/notebook.py /opt/conda/lib/python3.6/site-packages/tenso
 
 # Set backend for matplotlib
 ENV MPLBACKEND "agg"
+
+# geospatial learn course
+RUN pip install rtree && \
```

Comment:
Could you move this up to line 494?
https://github.com/Kaggle/docker-python/blob/abf2e82f4c60549ec055c553b524fcbcf1074843/Dockerfile#L494

Reply:
done! thanks